### ISSUE TITLE
Question regarding pixel_buffer size

### ISSUE THREAD

**Issue by perencia** [angmouzakitis/ZedBoard-AXI-VGA#4]
Shouldn't 
`                if h_cnt(5 downto 0) = 53 then
`
be 
`                if h_cnt(5 downto 0) = `**6**`3 then
`
?

I say it because later  you do 

`                next_address <= next_address + 128;
`

which is 64 * 32, the size of pixel_buffer.

Am I missing something?

Thanks!

**Comment by angmouzakitis** [comment]
Yes, 63 is the correct value, but i observed flickering and missing pixels if i had it to 63 because of the following: the clock for the 'axi_read_fsm' process is using the clock (M_AXI_ACLK) which is 102MHz. I don't remember about the the DDR clock but it was a factor of around 2-4 times higher. So, in order to get the next pixels i had to 'request' them some time before, because of the latency.
